google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -184,6 +185,8 @@ public String getDefaultValue() {
   private static final String USER_AGENT_PROPERTY_NAME = "userAgent";
   /** Query optimizer version to use for a connection. */
   private static final String OPTIMIZER_VERSION_PROPERTY_NAME = "optimizerVersion";
+  /** Query optimizer version to use for a connection. */
+  private static final String RETURN_COMMIT_STATS_PROPERTY_NAME = "returnCommitStats";
```

Comment:
The constant is less clear than the string itself and doesn't help. It's only used in one obvious place.

Reply:
It's used in two places (in the `parseReturnCommitStats` and to create the set of valid properties). But it's not something that can change in the future, as it would break existing connection URLs. I would prefer to change this in a separate PR, so the entire list of constants can be changed at once, instead of having one parameter that handled differently from the rest.

Reply:
Done